lib/generators/revise_auth/templates/migration.rb

```diff
@@ -0,0 +1,17 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
```

Comment:
Oh, we should remove these.

I was adding them for testing customizing the params in the `test/dummy` app but we don't need to require them.

Maybe we just comment them out with a note that you can add your own? Or we can do like Devise does and allow you to add your own columns when you run the generator?

Reply:
@excid3 how does devise handle this. Can something like this work?

Also thought of an optional argument that defaults to `User` for the model name

```
rails g revise_auth:install Account -a first_name:string last_name:string
```

Reply:
I believe it works by inheriting from the NamedBase generator. https://github.com/heartcombo/devise/blob/main/lib/generators/devise/devise_generator.rb#L7

The NamedBase is what handles the Class name as the first argument if I remember right.

Their ActiveRecord generator is run through the `hook_for :orm` and tehy have attributes as an argument here:
https://github.com/heartcombo/devise/blob/main/lib/generators/active_record/devise_generator.rb#L9

You can also look at the Rails model generator for examples.

We might also want to rename this to `revise_auth:model` which would be more descriptive.

```
rails g revise_auth:model Account first_name:string last_name:string
```

Reply:
Actually, Noticed also has an example that supports this same thing:

https://github.com/excid3/noticed/blob/master/lib/generators/noticed/model_generator.rb